src/index.ts

```diff
@@ -33,54 +23,59 @@ async function getWeather(location: string): Promise<WeatherData> {
   return weatherData;
 }
 
-async function useChatGptApi(prompt: string): Promise<void> {
+export async function useChatGptApi(prompt: string) {
```

Comment:
Still planning to move this to the test file?

Reply:
I imported the method to the test file. So it has been moved. I did not to push the whole file, just the method that triggers the use of the logger and returns the logentry

Reply:
I meant to move the entire contents of this file to spec (and delete this file).